content/en/tracing/trace_collection/compatibility/python.md

```diff
@@ -22,16 +22,16 @@ Two release branches are supported:
 
 | Release    | Support level        |
 |------------|----------------------|
-| `<1`       | Maintenance           |
-| `>=1.0,<2` | General Availability |
+| `<2`       | Maintenance           |
+| `>=2.0,<3` | General Availability |
 
 And the library supports the following runtimes:
 
 | OS      | CPU                   | Runtime | Runtime version | Support ddtrace versions |
 |---------|-----------------------|---------|-----------------|--------------------------|
-| Linux   | x86-64, i686, AArch64 | CPython | 2.7, 3.5-3.11   | `<2`                     |
```

Comment:
Perhaps the 2.7 reference since there will still be applications using dd-trace-py 0.x and 1.x, but should note that it's only for `<2`?

Reply:
Ah, good point.